config/importmap.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Pin npm packages by running ./bin/importmap
```

Comment:
When/how often should we be doing this? I guess I'm fuzzy on the new importmap + sprockets combo, mostly in terms of workflow but also in terms of conventions (see prior question about discrepancy between bootstrap js and css). Not blocking the PR, just hoping to understand more before we merge this.

Reply:
you do this when you add a JS dependency. This is the alternative to bunding the js using node. It transfers this work to the client, which makes the app simpler to deploy.